apps/web/src/components/(data)/model/playground/ModelPlayground.tsx

```
@@ -1626,12 +1958,23 @@ export default function ModelPlayground({
 			const message =
 				requestError instanceof Error
 					? requestError.message
-					: "Audio request failed. Please try again.";
+					: action === "music"
+						? "Music request failed. Please try again."
+						: "Audio request failed. Please try again.";
+			if (!isMountedRef.current) return;
 			setAudioError(message);
 		} finally {
+			musicPollControllerRef.current = null;
+			if (!isMountedRef.current) return;
```

Biome (2.4.13)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)